sdk/keyvault/keyvault-admin/src/lro/selectiveRestore/operation.ts

```diff
@@ -0,0 +1,288 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { AbortSignalLike } from "@azure/abort-controller";
+import { PollOperationState, PollOperation } from "@azure/core-lro";
+import {
+  OperationOptions,
+  operationOptionsToRequestOptionsBase,
+  RequestOptionsBase
+} from "@azure/core-http";
+import { KeyVaultClient } from "../../generated/keyVaultClient";
+import {
+  KeyVaultClientFullBackupStatusResponse,
+  KeyVaultClientSelectiveKeyRestoreOperationOptionalParams,
+  KeyVaultClientSelectiveKeyRestoreOperationResponse
+} from "../../generated/models";
+import { createSpan, setParentSpan } from "../../tracing";
+
+/**
+ * An interface representing the publicly available properties of the state of a restore Key Vault's poll operation.
+ */
+export interface SelectiveRestoreOperationState extends PollOperationState<undefined> {
```

Comment:
For our operation models we don't have `State` at the end

Reply:
Also we're only exposing `status`, `status_details`, `error`, `start_time`, `end_time`, and `job_id`

Reply:
this is the same for all of the models, excpet `BackupOperation` (or `BackupOperationState` for you), where we also have property `azure_storage_blob_container_uri`

Reply:
I can align with that, I think. I'll update this in a bit.

Reply:
I did update this. I'm resolving!